From 3b26258325b69c034b486fbdd12aa7d60b379cf6 Mon Sep 17 00:00:00 2001
From: Ignacio Casal Quinteiro <icq@gnome.org>
Date: Fri, 02 Jul 2010 19:17:00 +0000
Subject: Build fails without X

Thanks to Matthias Clasen for pointing out how to fix it.
---
diff --git a/gedit/gedit-command-line.c b/gedit/gedit-command-line.c
index db052a2..3134403 100644
--- a/gedit/gedit-command-line.c
+++ b/gedit/gedit-command-line.c
@@ -335,15 +335,13 @@ gedit_command_line_parse (GeditCommandLine   *command_line,
 	/* Setup command line options */
 	context = g_option_context_new (_("- Edit text files"));
 	g_option_context_add_main_entries (context, options, GETTEXT_PACKAGE);
-	g_option_context_add_group (context, gtk_get_option_group (FALSE));
+	g_option_context_add_group (context, gtk_get_option_group (TRUE));
 	g_option_context_add_group (context, egg_sm_client_get_option_group ());
 
 #ifdef ENABLE_INTROSPECTION
 	g_option_context_add_group (context, g_irepository_get_option_group ());
 #endif
 
-	gtk_init (argc, argv);
-
 	if (!g_option_context_parse (context, argc, argv, &error))
 	{
 		g_print(_("%s\nRun '%s --help' to see a full list of available command line options.\n"),
--
cgit v0.8.3.1
